docs/encryption.md

```diff
@@ -0,0 +1,13 @@
+# Encryption and Encrypted File Systems with The Sleuth Kit
+
+* We supposedly have encryption support for APFS and NTFS volume level encyrption.
+* We also supposeduly have support for FAT32 with EFS file level encryption. 
+
+The APFS encryption pipeline is relatively confusing. The standard usage includes first calling `mmls` on the disk image, however, I cannot get this to work on encrypted filesystems. 
+
```

Comment:
This comment is confusing because it's not clear who _I_ refers to . Can you rewrite this so that it objectively describes what is happening without the first person narrative? If you find something confusing, please put that in a separate section with the heading: "Issues requiring resolution."